libsolidity/ast/Types.h

```diff
@@ -694,11 +692,36 @@ class BoolType: public Type
 	TypeResult interfaceType(bool) const override { return this; }
 };
 
+/**
+ * Base class for types which can be thought of as several elements of other types put together.
+ * For example a struct is composed of its members, an array is composed of multiple copies of its
+ * base element and a mapping is composed of elements its value type (note that keys are not
```

Comment:
```suggestion
 * base element and a mapping is composed of elements of its value type (note that keys are not
```

Looks like it was a mistake in my original suggestion. Sorry for that :)

Reply:
Slightly [rephrased](https://github.com/ethereum/solidity/pull/9818/files#diff-c903efe2e4a71d75dc2eed4252cb24d8R698) to avoid "of - of".

Reply:
I think that the version with double `of` was easier to parse but that might be subjective. It's probably fine either way.